webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric.js

```diff
@@ -118,9 +121,19 @@ export class SakaiRubric extends RubricsElement {
             :
             ""
           }
+          ${this.exportPdf=='true' ? html`
+          <div class="action-container">
```

Comment:
Weird logic and using strings instead of boolean.

As Adrian mentioned do not use the "? X : Y" logic, just use Litelement templates.

Reply:
Done.